build.gradle

```diff
@@ -210,136 +210,136 @@ test {
-    // Cluster shrink exception thrown if we try to set numberOfNodes to 1, so only apply if > 1
-    if (_numNodes > 1) numberOfNodes = _numNodes
-    // When running integration tests it doesn't forward the --debug-jvm to the cluster anymore
-    // i.e. we have to use a custom property to flag when we want to debug elasticsearch JVM
-    // since we also support multi node integration tests we increase debugPort per node
-    if (System.getProperty("opensearch.debug") != null) {
-        def debugPort = 5005
-        nodes.forEach { node ->
-            node.jvmArgs("-agentlib:jdwp=transport=dt_socket,server=n,suspend=y,address=*:${debugPort}")
-            debugPort += 1
-        }
-    }
-    plugin(project.tasks.bundlePlugin.archiveFile)
-
-    plugin(provider(new Callable<RegularFile>(){
-        @Override
-        RegularFile call() throws Exception {
-            return new RegularFile() {
-                @Override
-                File getAsFile() {
-                    if (new File("$project.rootDir/$js_resource_folder").exists()) {
-                        project.delete(files("$project.rootDir/$js_resource_folder"))
-                    }
-                    project.mkdir js_resource_folder
-                    ant.get(src: job_scheduler_build_download,
-                            dest: js_resource_folder,
-                            httpusecaches: false)
-                    return fileTree(js_resource_folder).getSingleFile()
-                }
-            }
-        }
-    }))
-
-    // As of ES 7.7.0 the opendistro-anomaly-detection plugin is being added to the list of plugins for the testCluster during build before
-    // the opensearch-job-scheduler plugin, which is causing build failures. From the stack trace, this looks like a bug.
-    //
-    // Exception in thread "main" java.lang.IllegalArgumentException: Missing plugin [opensearch-job-scheduler], dependency of [opendistro-anomaly-detection]
-    //       at org.opensearch.plugins.PluginsService.addSortedBundle(PluginsService.java:452)
-    //
-    // One explanation is that ES build script sort plugins according to the natural ordering of their names.
-    // opendistro-anomaly-detection comes before opensearch-job-scheduler.
-    //
-    // The following is a comparison of different plugin installation order:
-    // Before 7.7:
-    // ./bin/elasticsearch-plugin install --batch file:opendistro-anomaly-detection.zip file:opensearch-job-scheduler.zip
-    //
-    // After 7.7:
-    // ./bin/elasticsearch-plugin install --batch file:opensearch-job-scheduler.zip file:opendistro-anomaly-detection.zip
-    //
-    // A temporary hack is to reorder the plugins list after evaluation but prior to task execution when the plugins are installed.
-    nodes.each { node ->
-        def plugins = node.plugins
-        def firstPlugin = plugins.get(0)
-        plugins.remove(0)
-        plugins.add(firstPlugin)
-    }
-}
+//task integTest(type: RestIntegTestTask) {
```

Comment:
When commenting out blocks of code please add a comment and link to whatever issue is tracking the incompatibility.  When someone comes across this in the future and asks "why is this commented out" they should be able to see if whatever the problem was has been resolved.